test/BaselineOutput/Common/Onnx/BinaryClassification/BreastCancer/OneHotBagPipeline.txt

```diff
@@ -324,8 +324,8 @@
           {
             "name": "target_weights",
             "floats": [
-              0.50476193,
-              -0.97911227
+              0.504761934,
+              -0.979112267
```

Comment:
How come these changed?

Reply:
Not sure. But I see this occurring off and on that the baselines numbers change when we run them locally. I am ignoring them because the change is in the 7th decimal place

---
In reply to: [381858622](https://github.com/dotnet/machinelearning/pull/4857#discussion_r381858622) [](ancestors = 381858622)